### ISSUE TITLE
Structured output from toolbelt commands, for use in scripts

### ISSUE THREAD

**Issue by slepkin** [heroku/heroku-pg#104]
As mentioned in https://github.com/heroku/cli/issues/42, it's useful to be able to output JSON (or any other standardized structure) when a command is run in the CLI. Many CLI commands have `--json` as an option, but `pg:info` does not. I envision it like this:

```
$heroku pg:info
{
    "FOLLOWER_DATABASE": {
        "Plan": "Premium 2",
        "Following": "DATABASE",
        ...
    },
    "DATABASE": {
        "Plan": "Premium 4",
        ...
    }
}
```

**Comment by msakrejda** [comment]
I agree this would be useful, but I don't think we can offer a stable output format here without stabilizing and formalizing the internal APIs we use to get this info. That's something we're definitely interested in, but we don't have a timeline right now.
